component/pom.xml

```diff
@@ -562,6 +576,7 @@
                             org.apache.kafka.connect.*,
                             org.apache.kafka.common.*,
                             org.apache.kafka.clients.*,
+                            org.antlr.*,
```

Comment:
Any specific reason to pack antlr?

Reply:
"java.lang.ClassNotFoundException: org.antlr.v4.runtime.CharStreams cannot be found by siddhi-io-cdc_2.0.5" is thrown